tests/Sg/ConsultantInscriptionIntegrationTest.php

```diff
@@ -0,0 +1,644 @@
+
+    public function testGetConsultantInscriptionDocumentShouldReturn404()
+    {
+        $filename = 'documentIdentity';
+
+        $env = Environment::mock([
+            'REQUEST_METHOD' => 'GET',
+            'REQUEST_URI' => "/api/v1/sg/consultant-inscription/2/document/$filename",
+        ]);
+        $req = Request::createFromEnvironment($env);
+        $this->app->getContainer()['request'] = $req;
+        $response = $this->app->run(false);
+        $this->assertSame(404, $response->getStatusCode());
+    }
+
+    /**
+     * @throws MethodNotAllowedException
+     * @throws NotFoundException
+     */
+    public function testValidateConsultantInscriptionShouldReturn201()
+    {
+        $filetypes = array('identity', 'residence_permit', 'rib', 'scolary_document', 'vitale_card', 'cvec');
+        $filepaths = array();
+        foreach ($filetypes as $fileType) {
+            $file_path = "documents/inscription/$fileType/test.pdf";
+            array_push($filepaths, $file_path);
+            $temp_file = FileHelper::makeNewFile(FileHelper::normalizePath($file_path));
+            FileHelper::closeFile($temp_file);
+        }
+
+        $env = Environment::mock([
+            'REQUEST_METHOD' => 'POST',
+            'REQUEST_URI' => '/api/v1/sg/consultant-inscription/1/validate',
+        ]);
+        $req = Request::createFromEnvironment($env);
+        $this->app->getContainer()['request'] = $req;
+        $response = $this->app->run(false);
+
+        $body = json_decode($response->getBody());
+
+        $this->assertSame(201, $response->getStatusCode());
+        $this->assertNotNull($body->id);
+        $this->assertSame('Bruce.Wayne', $body->username);
+        $this->assertSame("Bruce", $body->firstName);
+        $this->assertSame("Wayne", $body->lastName);
+        $this->assertSame(1, $body->gender->id);
+        $this->assertSame("bruce.wayne@batman.com", $body->email);
+        $this->assertSame("2000-02-14", $body->birthday);
+        $this->assertSame(3, $body->department->id);
+        $this->assertSame("0033123456789", $body->telephone);
+        $this->assertSame(null, $body->profilePicture);
+        $this->assertNotNull($body->address->id);
+        $this->assertSame('13 Rue du renard', $body->address->line1);
+
+        foreach ($filepaths as $file_path) {
+            FileHelper::deleteFile(FileHelper::normalizePath($file_path));
+        }
+    }
+
+    /**
+     * @throws MethodNotAllowedException
+     * @throws NotFoundException
+     */
+    public function testValidateConsultantInscriptionShouldReturn404()
+    {
+        $env = Environment::mock([
+            'REQUEST_METHOD' => 'POST',
+            'REQUEST_URI' => '/api/v1/sg/consultant-inscription/1000/validate',
+        ]);
+        $req = Request::createFromEnvironment($env);
+        $this->app->getContainer()['request'] = $req;
+        $response = $this->app->run(false);
+
+        $this->assertSame(404, $response->getStatusCode());
+    }
+}
```

Comment:
Pas grand chose à dire sur les tests, juste j'en avait fait un où on tentait de valider 2 fois une même inscription.

Reply:
Compliqué de faire ça car je retourne le membre crée dans la réponse de la requête. Si je fais une deuxième requête il me donne erreur genre 'invalid character in output buffer' (la même que tu avais avec le delete all members). J'ai testé ce cas à la main avec postman